### DISCUSSION TITLE
Is there any gerber file here?

### DISCUSSION THREAD

**Discussion by RomanNpolice** [ElectroTechnique/electrotechnique.github.io#45] [Hardware Issues]
Recently, I became interested in Arduino synth and I just found it and make it myself. So I read documents of it and acknowledge there are no gerber files to make pcb on it. I googled myself to find Tsynth's gerber files and pcb file. But I couldn't find it.
Where can i found it or order pcbs on internet? Can you add gerber files to repositorys?

**Comment by Electrotechnique** [comment]
More details here: [electrotechnique.cc/](http://electrotechnique.cc/)
